modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -956,6 +976,22 @@ func (suite *KeeperTestSuite) TestQueryPacketAcknowledgement() {
 			},
 			false,
 		},
+		{
+			"ack not found",
+			func() {
+				path := ibctesting.NewPath(suite.chainA, suite.chainB)
+				path.Setup()
+				expAck = []byte("hash")
+				suite.chainA.App.GetIBCKeeper().ChannelKeeper.SetPacketAcknowledgement(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, 1, expAck)
+
+				req = &types.QueryPacketAcknowledgementRequest{
+					PortId:    "test-port-id",
+					ChannelId: "test-channel-id",
+					Sequence:  2,
+				}
+			},
+			false,
+		},
```

Comment:
This is similar to the "commitment" test. In the sense that the one below (channel not found) was actually hitting the line that checks for the ack, so now that we check for the channel before that, we need to cover this scenario in a slightly more verbose test.